prog-spe.js

```diff
@@ -0,0 +1,113 @@
+const speakersArr = [
+  {
+    id: 0,
+    img: 'project-image/trainer-1.jpg',
+    alt: 'abel',
+    speakerName: 'Abel Fikre',
+    speakerStatus: 'Agronomist, Breeder, Designer and Software Developer',
+    speakerInfo: 'Lorem ipsum dolor sit amet.',
+  },
+  {
+    id: 1,
+    img: 'project-image/trainer-2.jpg',
+    alt: 'tade',
+    speakerName: 'Tadele Warrie',
+    speakerStatus: 'Seamstress and Designer',
+    speakerInfo: 'Lorem ipsum dolor sit amet.',
+  },
+];
+
+let speakers1 = '';
+
+speakersArr.forEach((speaker) => {
+  speakers1 += `<li class="speakers">
+  <div class="img-card">
+      <img src="${speaker.img}" alt="${speaker.alt}">
+  </div>
+  <div class="img-desc">
+      <h3 class="speaker-name">${speaker.speakerName}</h3>
+      <p class="speaker-status">${speaker.speakerStatus}</p>
+      <div class="h-r3"></div>
+      <p class="speaker-info">${speaker.speakerInfo}</p>
+  </div>
+  </li>`;
+});
+
+const speakers = document.querySelector('.speakers-list');
+speakers.innerHTML = speakers1;
+
+const speakersArr2 = [
+  {
+    id: 0,
+    img: 'project-image/trainer-3.jpg',
+    alt: 'jarmi.',
+    speakerName: 'Jarmia Bayisa',
+    speakerStatus: 'Field Agronomist, Seamster and Designer',
+    speakerInfo: 'Lorem ipsum dolor sit amet.',
+  },
+  {
+    id: 1,
+    img: 'project-image/trainer-4.jpg',
+    alt: 'mamp',
+    speakerName: 'Mamo fidano',
+    speakerStatus: 'Electrical Engineer, Seamster and Designer',
+    speakerInfo: 'Lorem ipsum dolor sit amet.',
+  },
+];
+
+let speakers12 = '';
+
+speakersArr2.forEach((speaker2) => {
+  speakers12 += `<li class="speakers">
+  <div class="img-card">
+      <img src="${speaker2.img}" alt="${speaker2.alt}">
+  </div>
+  <div class="img-desc">
+      <h3 class="speaker-name">${speaker2.speakerName}</h3>
+      <p class="speaker-status">${speaker2.speakerStatus}</p>
+      <div class="h-r3"></div>
+      <p class="speaker-info">${speaker2.speakerInfo}</p>
+  </div>
+  </li>`;
```

Comment:
- Great job for making the speaker list dynamic, I think it would be better if you wrap each speakers with `article` for sematic reasons 👍